sources/packages/forms/src/form-definitions/exportfinancialreports.json

```diff
@@ -501,87 +161,21 @@
                         "minuteIncrement": 5,
                         "time_24hr": false,
                         "saveAs": "text",
-                        "locale": "en"
+                        "locale": "en",
+                        "displayInTimezone": "viewer"
                       },
-                      "inputMask": "",
-                      "displayMask": "",
-                      "allowMultipleMasks": false,
-                      "customClass": "",
-                      "tabindex": "",
-                      "autocomplete": "",
-                      "hidden": false,
-                      "hideLabel": false,
-                      "showWordCount": false,
-                      "showCharCount": false,
-                      "mask": false,
-                      "autofocus": false,
-                      "spellcheck": true,
-                      "disabled": false,
                       "tableView": true,
-                      "modalEdit": false,
-                      "multiple": false,
-                      "persistent": true,
-                      "inputFormat": "plain",
-                      "protected": false,
-                      "dbIndex": false,
-                      "case": "",
-                      "truncateMultipleSpaces": false,
-                      "encrypted": false,
-                      "redrawOn": "",
-                      "clearOnHide": true,
-                      "customDefaultValue": "",
-                      "calculateValue": "",
-                      "calculateServer": false,
-                      "allowCalculateOverride": false,
-                      "validateOn": "change",
                       "validate": {
                         "required": true,
-                        "minLength": "",
-                        "maxLength": "",
-                        "minWords": "",
-                        "maxWords": "",
-                        "pattern": "",
-                        "customMessage": "",
-                        "custom": "valid = data.params.startDate\r\n  ? data.params.startDate < data.params.endDate\r\n    ? true\r\n    : \"End Date must be after Start Date\"\r\n  : \"Please provide Start date\";",
-                        "customPrivate": false,
-                        "json": "",
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false
+                        "custom": "valid = data.params.startDate\r\n  ? data.params.startDate < data.params.endDate\r\n    ? true\r\n    : \"End Date must be after Start Date\"\r\n  : \"Please provide Start date\";"
```

Comment:
Should this one allow start and end dates to be the same in case the user wants to select only one day?

Reply:
I believe this was a part of the initial requirement because I see that the error message for the validation `End Date must be after Start Date` was pre-written.

Reply:
Even if it makes sense it would be optional since it is not part of this report implementation.
If you change it, please check other reports to ensure they would also work as expected.
If all are using `between` it should be ok.

Reply:
It might be worth letting Biz know that they won't be able to select just one day.

Reply:
@andrepestana-aot agreed, but this discussion should not block this PR, to be clear.